src/qt/clientmodel.h

```diff
@@ -81,6 +81,10 @@ class ClientModel : public QObject
     QString formatClientStartupTime() const;
     QString dataDir() const;
 
+    // get/set state about autorequesting-blocks during IBD
+    bool isAutorequestBlocks() const;
```

Comment:
Nit: `isAutorequestingBlocks` (add `ing`). The `setAutorequestBlocks` below is fine as is, though.